### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by vext01**
See individual commits.

### CODE REVIEW

File: src/tech/yk_structure.md [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
 will never be traced and therefore we can (and should) optimise this code.
 Further we want to be able to optimise the dependencies of the JIT runtime.
 
-To this end, we have two Rust workspaces in the `yk` repo: the "internal"
-(optimised) workspace, and the "external" (unoptimised) workspace. The external
-workspace then talks to the internal workspace via `extern` functions defined
-in the `ykshim` crate.
+To this end, we have three Rust workspaces in the `yk` repo:
+ - The top-level xtask workspace: our build system.
+ - The untraced workspace: the "always optimised" JIT runtime.
```
Comment (ltratt) [inline src/tech/yk_structure.md:15]: Should we put `untraced` and `traced` in backticks here (and elsewhere in this document)?

Comment (vext01) [reply src/tech/yk_structure.md]: Like so: 59103b5

Comment (ltratt) [reply src/tech/yk_structure.md]: Excellent!

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**